### PR TITLE
Docs converted from to markdown, updates for bundler v1.10 and some minor changes.

### PR DESCRIPTION
**Pull request by rgb-one**

### CODE REVIEW

File: config.rb [hunk 1]
```diff
@@ -1,9 +1,28 @@
...
 
+# Syntax highlighting
+activate :syntax
+
+# Set HAML to render HTML5 by default. It's important that HAML outputs "ugly" HTML to not mess with code blocks
+set :haml, format: :html5, ugly: true
+
+# Set markdown features for Redcarpet
+set :markdown_engine, :redcarpet
+set :markdown,
```
Comment (segiddins) [inline config.rb]: some of these are duplicated

File: config.rb [hunk 2]
```diff
@@ -1,9 +1,28 @@
...
-set :markdown_engine, :kramdown
-
+# Bundler versions
 set :versions, `rake versions`.split
 set :current_version, versions.last
 
+# Syntax highlighting
+activate :syntax
+
+# Set HAML to render HTML5 by default. It's important that HAML outputs "ugly" HTML to not mess with code blocks
```
Comment (segiddins) [inline config.rb]: please link to the HAML issue that's tracking this?

File: source/contributors.haml [hunk 3]
```diff
@@ -1,4 +1,9 @@
-%p== Development of Bundler and related projects is managed by the Bundler Core Team, led by André Arko. Bundler was originally written by Yehuda Katz and Carl Lerche. Additional support and code has been provided by #{link_to 'over 300 other contributors, as listed on Github.', 'http://github.com/bundler/bundler/contributors'}
+%p
+
+  Development of Bundler and related projects is managed by the 
+  Bundler Core Team, led by André Arko. Bundler was originally written by 
+  Yehuda Katz and Carl Lerche. Additional support and code has been provided by 
+  #{link_to 'over 300 other contributors, as listed on Github.', 'http://github.com/bundler/bundler/contributors'}
```
Comment (segiddins) [inline source/contributors.haml]: `over 400`

File: source/layouts/blog_layout.haml [hunk 4]
```diff
@@ -1,5 +1,5 @@
 !!!
-- v = current_page.url.scan(/v\d\.\d/).first || current_version
+- v = current_page.url.scan(/v\d\.\d\d/).first || current_version
```
Comment (segiddins) [inline source/layouts/blog_layout.haml]: `v\d+\.\d+`

File: source/contributors.haml [hunk 5]
```diff
@@ -1,4 +1,9 @@
-%p== Development of Bundler and related projects is managed by the Bundler Core Team, led by André Arko. Bundler was originally written by Yehuda Katz and Carl Lerche. Additional support and code has been provided by #{link_to 'over 300 other contributors, as listed on Github.', 'http://github.com/bundler/bundler/contributors'}
+%p
```
Comment (segiddins) [inline source/contributors.haml]: why is this not markdown?

File: source/layouts/layout.haml [hunk 6]
```diff
@@ -1,5 +1,5 @@
 !!!
-- v = current_page.url.scan(/v\d\.\d/).first || current_version
+- v = current_page.url.scan(/v\d\.\d\d/).first || current_version
```
Comment (segiddins) [inline source/layouts/layout.haml]: ^ same

File: source/stylesheets/code.css.scss [hunk 7]
```diff
@@ -1,4 +1,4 @@
-pre.highlight { padding: 10px; font-size: 13px; font-weight: 400; font-family: 'Source Code Pro', Monaco, monospace; }
+pre.highlight { }
```
Comment (segiddins) [inline source/stylesheets/code.css.scss]: no need for the empty block

File: config.rb (hunk 2 above)
Comment (rgb-one) [inline config.rb]: https://github.com/middleman/middleman-syntax#indentation-problems

**Comment by indirect** [comment]
This is really close! I can't merge it quite yet because the link buttons have all stopped being buttons. Here's how it looks on the current live site:

screen shot 2015-06-24 at 9 35 01 pm

Here's how it looks in this pull request:

screen shot 2015-06-24 at 9 35 06 pm

Almost there. :)

**Comment by rgb-one** [comment]
Haha, I switched to redcarpet as markdown engine but kramdown offers functionality that can make the buttons possible. I will send a pull request soon reflecting this change.

**Comment by indirect** [comment]
Cool! You don't have to open a new branch or a new PR, just pull the devel branch, add commits, and then push.

**Comment by rgb-one** [comment]
The buttons now show in the latest revision of the markdown docs. I will close this older version of the markdown and send another pull request with the updated markdown docs.

**Pull request closed without merging**